docs/developer-docs/daos/nns/proposal-voting.mdx

```diff
@@ -0,0 +1,174 @@
+This can be suitable for individuals who operate a known neuron or for organizations where one person is trusted with this task.
+
+:::
+
+### Create a community neuron
+
+Additional information how to create a community neuron will follow soon!
+
+
+### Present your neuron to the community
+A known neuron needs a name and a description. Often the owners of known neurons use the description part to present themselves a little.
+
+There are no rules of course, but it can make sense for neuron owners to present themselves to the community already before submitting
+ the proposal to register the known neuron. This can serve as a way for promoting a neuron and explaining the motivation behind it.
+This can, for example, be done with a forum post or by sharing information on social media, such as OpenChat or X.
+It might be useful to share the following information with others, so that followers can make well-informed decisions who to follow for which topic:
+* **Who you are**: Name of the person or group and their relationship to the ICP. You may want to argue why the people behind the neuron have a long-term interest in the wellbeing of the ICP and are therefore motivated to make good voting decisions.
+* **Neuron control**: How controls the neuron? Does it have a single controller or can it only be changed by a group of people?
+* **Covered topics**: A description of what topics the known neuron will vote on. Does the neuron commit to always vote on these topics?
+Since following is done based on proposal topics, this is relevant information for followers.
+Usually, neurons would want to follow other neurons neurons that always vote on a given topic in order not to miss out on [voting rewards](./staking-voting-rewards.mdx).
+* **Depth of reviews**: A description of how the known neuron plans to verify proposals. To which depth will it review the code or the content of the proposal and how is this done?
+* **Following setting**: For the topics where the know neuron does not vote directly, who does the neuron follow?
+* **Voting principles**: What are the principles according to which you plan to vote.
+* **Link to the neuron ID / proposal**: If you already created the neuron, you may want to provide its ID or even link to a dashboard where it can be inspected. The same holds for the proposal to register it as a know neuron in the case where the proposal was already made.
+
+:::info
+There are no rules how a know neuron has to vote. However, due to the above reasons, it might be useful for known neurons to **focus on fewer topics and commit to always vote on those** rather than trying to cover a lot of topics and vote less reliably. For example, they could become experts on some proposal topics. If a known neuron commits to always vote on a certain topic it might be more likely that others follow the neuron on this topic as the followers want to make sure they are not missing out on rewards.
```

Comment:
```suggestion
There are no rules regarding how a known neuron has to vote. However, due to the above reasons, it might be useful for known neurons to **focus on fewer topics and commit to always vote on those** rather than trying to cover a lot of topics and vote less reliably. For example, they could become experts on some proposal topics. If a known neuron commits to always vote on a certain topic, it might be more likely that others follow the neuron on this topic as the followers want to make sure they are not missing out on rewards.
```